PowerArubaSW/Public/VSF.ps1

```diff
@@ -0,0 +1,182 @@
+#
+# Copyright 2018, Alexis La Goutte <alexis.lagoutte at gmail dot com>
+# Copyright 2018, C�dric Moreau <moreaucedric0 at gmail dot com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+function Set-ArubaSWVsfDisable {
+
+    <#
+        .SYNOPSIS
+        Set Vsf Disable on ArubaOS Switch.
+
+        .DESCRIPTION
+        Set Vsf Disable on ArubaOS Switch.
+
+        .EXAMPLE
+        Set-ArubaSWVsfDisable
+        Set the vsf disable on the switch.
+    #>
+
+    Begin {
+    }
+
+    Process {
+
+        $url = "rest/v4/stacking/vsf/disable"
+
+        $response = invoke-ArubaSWWebRequest -method "POST" -url $url -body " "
```

Comment:
need also to add -connection $connection info (for support [multi connection](https://github.com/PowerAruba/PowerArubaSW#MultiConnection))

Look other file..